static/src/javascripts/projects/common/modules/commercial/creatives/template-preprocessor.js

```diff
@@ -43,6 +54,41 @@ define([
     var manualContainerCtaSoulmatesTpl;
     var manualContainerCtaMembershipTpl;
 
+    function preprocessLogo(tpl) {
```

Comment:
Do you mean that these values will soon be set in an actual DFP creative template?